x-pack/legacy/plugins/siem/server/lib/detection_engine/signals/build_rule.ts

```diff
@@ -55,5 +55,7 @@ export const buildRule = ({
     created_by: createdBy,
     updated_by: updatedBy,
     threats: ruleParams.threats,
+    created_at: undefined,
+    updated_at: undefined,
```

Comment:
Is this right? Didn't you want to copy over the value for the GET?
```ts
created_at: ruleParams.createdAt
updated_at: ruleParams.updatedAt
```

Reply:
Yup I should change this.